scripts/gh/seed-overview-depths.sh

```
@@ -0,0 +1,42 @@
+if [[ -z "$proj_id" ]]; then
+  proj_id=$(gh api graphql -f query='query($login:String!){organization(login:$login){projectsV2(first:50){nodes{id title}}}}' -F login="$owner" --jq '.data.organization.projectsV2.nodes[] | select(.title=="TASKS + SLAPS OVERVIEW") | .id')
+fi
+
+fields=$(gh api graphql -f query='query($id:ID!){node(id:$id){... on ProjectV2{fields(first:50){nodes{__typename ... on ProjectV2Field{ id name dataType }}}}}}' -F id="$proj_id" --jq '.data.node.fields.nodes')
+depth_id=$(echo "$fields" | jq -r '.[] | select(.name=="Depth") | .id')
+if [[ -z "$depth_id" || "$depth_id" == null ]]; then
+  echo "Depth field is missing on the overview project; create it in the UI or via GraphQL first." >&2
+  exit 1
+fi
+
+add_or_get_item(){ local num="$1"; local cid=$(gh issue view --repo "$owner/$repo" "$num" --json id --jq .id); gh api graphql -f query='mutation($proj:ID!,$content:ID!){ addProjectV2ItemById(input:{projectId:$proj, contentId:$content}){ item { id } } }' -F proj="$proj_id" -F content="$cid" --jq '.data.addProjectV2ItemById.item.id' 2>/dev/null || true; }
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)